src/02-common-principles.md

```diff
@@ -33,10 +33,10 @@ clarify them here.
     scanning sequence/protocol.
 
 1.  Data type - a functional group of different types of data. In BIDS we define
-    five data types: func (task based and resting state functional MRI), dwi
+    six data types: func (task based and resting state functional MRI), dwi
     (diffusion weighted imaging), fmap (field inhomogeneity mapping data such as
     field maps), anat (structural imaging such as T1, T2, etc.), meg
-    (magnetoencephalography).
+    (magnetoencephalography), beh (behavioural).
```

Comment:
while at it may be fix those "incorrect" spellings:
```
$> git grep behavioral
CHANGES.md:-   Added section on behavioral experiments with no accompanying MRI
src/04-modality-specific-files/05-behavioural-experiments.md:In addition to logs from behavioral experiments performed along imaging data
src/99-appendices/03-hed.md:HED is a controlled vocabulary of terms describing events in a behavioral
src/99-appendices/03-hed.md:all behavioral experiments. Each level of a hierarchical tag is delimited with
```
for consistency

Reply:
Thanks, good point. There are now 7x "behavioral" and 9x "behavioural" in the whole spec  (including my edits). The slightly shorter behaviour and behavior also both happen.

I suppose you prefer the American English, I actually don't care (as long as it is consistent). Is there a formal preference for spelling?